install.sh

```diff
@@ -254,4 +276,8 @@ echo "### Installing myst & dependencies"
 install
 echo "### Installing myst & dependencies - done"
 
+echo "### Ensuring sane paths for dependencies"
```

Comment:
What about "Normalizing dependency paths" or "Creating dependency symlinks"? Those sound more to the point to me.